event-publisher/src/main/java/uk/gov/ons/ctp/common/event/EventTopic.java

```diff
@@ -9,7 +9,10 @@
   SURVEY_LAUNCH("event_survey-launch", EventType.SURVEY_LAUNCH),
   REFUSAL("event_refusal", EventType.REFUSAL),
   UAC_UPDATE("event_uac-update", EventType.UAC_UPDATE),
-  CASE_UPDATE("event_case-update", EventType.CASE_UPDATE);
+  CASE_UPDATE("event_case-update", EventType.CASE_UPDATE),
+  SURVEY_UPDATE("event_survey-update", EventType.SURVEY_UPDATE),
+  COLLECTION_EXERCISE_UPDATE(
+      "event_collection-exercise-update", EventType.COLLECTION_EXERCISE_UPDATE); // TODO topic name
```

Comment:
please correct the topic name and remove the TODO comment.

Reply:
Topic name was already correct, TODO removed